applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,1770 @@
+{
```

Comment:
<p>We should consider including some explanation of the newton-raphson method, I think.</p><p> </p><p>Also, did you pull this code from somewhere? The use of comments and raising value errors feel out of style with the rest of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I need error raising in order to do an iterative solve to ensure that it ends at some point. I can simplify the comments and look at the styling.

Reply:
Does using [scipy.optimize.newton](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.newton.html) maybe help here?  Might give a place to point to more docs and abstract away some code.

Reply:
Yes definitely. I will update it to use this.